### PR TITLE
Changes from dypa implemented

### PR DESCRIPTION
**Pull request by zelf**
https://github.com/sonata-project/SonataAdminBundle/pull/1971

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
Why open a second PR for that?

File: Admin/AdminHelper.php [hunk 1]
```diff
@@ -37,25 +34,6 @@ public function __construct(Pool $pool)
...
 
     /**
-     * @throws \RuntimeException
-     *
-     * @param \Symfony\Component\Form\FormBuilderInterface $formBuilder
-     * @param string                                       $elementId
-     *
-     * @return \Symfony\Component\Form\FormBuilderInterface
-     */
-    public function getChildFormBuilder(FormBuilderInterface $formBuilder, $elementId)
```
Comment (Soullivaneuh) [inline Admin/AdminHelper.php]: Removing a public method is completely BC break.

BTW, why removing it?

If it's necessary, please make BC with deprecation system.

File: Admin/AdminHelper.php [hunk 2]
```diff
@@ -101,88 +74,48 @@ public function getAdmin($code)
...
+     * @param AdminInterface $admin
+     * @param $elementId
+     * @throws \Exception
+     * @internal param object $object
+     * @internal param FieldDescriptionInterface $fieldDescription
      *
-     * @throws \RuntimeException
      */
-    public function addNewInstance($object, FieldDescriptionInterface $fieldDescription)
+    public function addNewInstance(AdminInterface $admin, $elementId)
```
Comment (Soullivaneuh) [inline Admin/AdminHelper.php]: Changing method signature is BC break too.

**Comment by zelf** [comment]
Sorry, I didn't mean to commit on the main project ! I need that changes for myself. and the PR is blocked. No way to make things move so I just wanted to have a working fork. 
Can you/I cancel that?

File: Admin/AdminHelper.php [hunk 3]
```diff
@@ -190,12 +123,20 @@ public function addNewInstance($object, FieldDescriptionInterface $fieldDescript
...
-     * @param string $property
-     *
+     * @param AdminInterface $admin
+     * @param $elements
      * @return string
+     * @internal param string $property
+     *
      */
-    public function camelize($property)
+    protected function entityClassNameFinder(AdminInterface $admin, $elements)
```
Comment (Soullivaneuh) [inline Admin/AdminHelper.php]: BC break. Same reasons.

**Comment by Soullivaneuh** [comment]
> Sorry, I didn't mean to commit on the main project ! I need that changes for myself. and the PR is blocked. No way to make things move so I just wanted to have a working fork. Can you/I cancel that?

@zelf got it.

**Pull request closed without merging**